tests/app/v2/broadcast/test_post_broadcast.py

```diff
@@ -103,12 +103,14 @@ def test_valid_post_cap_xml_broadcast_returns_201(
     assert response_json['personalisation'] is None
     assert response_json['service_id'] == str(sample_broadcast_service.id)
 
-    # TEMPORARY: while we repurpose "areas"
```

Comment:
The comment here has been removed, but no related code which I guess is just because that has already been deleted? This test does have a line repeated twice though (`assert response_json['areas']['simple_polygons'][0][-1] == [53.10562, 0.244127]`)

Reply:
Good spot. I've amended the second commit to remove the extra line and explain both removals in the commit message.